style.css

```diff
@@ -1 +1,534 @@
 /*  Add your styling here */
+body {
+  background-color: red;
+}
+.head {
+  background-color: red;
+}
+.logo {
+  margin-top: 40px;
+}
+.contact-info {
+  background-color: skyblue;
+  text-align: center;
+  margin-left: 85%;
+}
+.list {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-around;
+  padding-left: 0%;
+}
+.list-item {
+  background-color: black;
+  width: 100%;
+  border: 1px solid red;
+  color: red;
+  padding: 10px;
+  text-align: center;
+}
+.main-container {
+  background-color: black;
+  color: red;
+}
+.sidebar {
+  width: 25%;
+  display: flex;
+  flex-direction: column;
+  float: left;
+}
+
+.main-image {
+  width: 75%;
+  float: right;
+  padding-top: 5px;
+  margin-bottom: 10px;
+}
+.title {
+  background-color: yellow;
+  color: black;
+  margin-bottom: 0;
+}
+.info {
+  background-color: blue;
+  margin: 30px;
+}
+carosel-container {
+  width: 100%;
+  height: 600px;
+  margin-left: auto;
+  margin-right: auto;
+  overflow: hidden;
+  text-align: center;
+}
+.image-carosel {
+  width: 700px;
+  height: 600px;
+  position: relative;
+  transition: left 2s;
+  display: flex;
+  z-index: 10;
+
+  -webkit-transition: left 2s;
+  -moz-transition: left 2s;
+  -o-transition: left 2s;
+}
+.image-set1 {
+  width: 100%;
+  float: left;
+  margin: 0;
+  padding: 0;
+}
+.button-container {
+  top: -20px;
+  position: relative;
+}
+.image-button {
+  display: inline-block;
+  height: 10px;
+  width: 10px;
+  border-radius: 5px;
+  background-color: white;
+}
+#image-set1-1:target ~ .image-carosel {
+  left: 0px;
+}
+
+#image-set1-1:target ~ #image2 {
+  display: none;
+  border: solid red 1px;
+}
+
+#image-set1-2:target ~ .image-carosel {
+  left: -800px;
+}
+#image-set1-3:target ~ .image-carosel {
+  left: -1600px;
+}
+#image-set1-4:target ~ .image-carosel {
+  left: -2250px;
+}
+.section-container {
+  width: 0;
+}
+.image-tag2 {
+  width: 100%;
+  display: flex;
+  list-style-type: none;
+}
+.image-set2 {
+  width: 250px;
```

Comment:
There are 4 images at 250px each, so all together they require at least 1000px wide. This is breaking the layout on small desktop.
A possible solution is to set each `li` to 25%, and the images inside to 100%.